src/Compilers/CSharp/Portable/Symbols/Synthesized/SynthesizedContainer.cs

```diff
@@ -15,6 +15,8 @@ namespace Microsoft.CodeAnalysis.CSharp.Symbols
     /// </summary>
     internal abstract class SynthesizedContainer : NamedTypeSymbol
     {
+        internal const string StructFieldCaptureName = "<struct-field-capture-name>";
```

Comment:
I think ```v``` is the next kind that's not taken.

---
In reply to: [134593614](https://github.com/dotnet/roslyn/pull/21664#discussion_r134593614) [](ancestors = 134593614,134590720)